07-github.md

```diff
@@ -140,17 +140,22 @@ Branch master set up to track remote branch master from origin.
 
 > ## Password Managers {.callout}
 >
-> If your operating system has a password manager configured, `git push` will
-> try to use it when it needs your username and password. If you want to type
-> your username and password at the terminal instead of using
-> a password manager, type:
+> Some systems will open a password manager when you run `git push`.  The
```

Comment:
I'm not sure about “Some systems will open…”.  It's really “Some systems will set `SSH_ASKPASS` in an attempt to be helpful, which will cause Git to open…”.  How about keeping @rgaiacs' original phrasing here and just adding a [Git for Windows][1] parenthetical?

> If your system has a password manager configured (like [Git for Windows][1] [sometimes does][2]), `git push` ….


[1]: https://git-for-windows.github.io/
[2]: https://github.com/git-for-windows/build-extra/blob/4c8d998fbb6a25e25fa16b8ba79e8efde2ffbdec/git-extra/env.sh#L4-L16